hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -378,7 +389,17 @@
            for (int j = 0; j < numMetaReplicas; j++) {
              RegionInfo meta = RegionReplicaUtil.getRegionInfoForReplica(
                                      RegionInfoBuilder.FIRST_META_REGIONINFO, j);
-             ServerName metaLocation = MetaTableLocator.waitMetaRegionLocation(master.getZooKeeper(), j, 1);
+             //If a metaLocation is null, All of its info would be empty here to be displayed.
+             ServerName metaLocation = null;
+             try {
+               metaLocation = MetaTableLocator.waitMetaRegionLocation(master.getZooKeeper(), j, 1);
+             } catch (NotAllMetaRegionsOnlineException e) {
+               //Since a region in transition state throw a NotAllMetaRegionsOnlineException then
+               //could cause the whole UI crash, and the operator could have learned that this
+               //region is in transition state from the master status UI. We should ignore to display
+               //this region info.
+               continue;
```

Comment:
> In this case, the user is looking at the meta table specifically. Maybe the below if (metaLocation != null) {... needs an else block that prints a message stating that some region location(s) could not be identified?

@ndimiduk  Since the user may learn the meta region is in transition state from the transition list in Master status UI， Should we still show any details here?

Reply:
After I check the current implementation it shows that If a region of user-level table is in transition, it does not be displayed in its table details page. So I think the meta region should be the same.